dts/riscv/snps/rmx5xx.dtsi

```diff
@@ -0,0 +1,63 @@
+/ {
```

Comment:
As it doesn't represent any real SoC/board probably it worth to move it to board directory (as we have it for nSIM classic)

Reply:
Done